frontend/lib/norent/letter-builder/know-your-rights.tsx

```diff
@@ -8,10 +8,19 @@ import {
   getUSStateChoiceLabels,
   USStateChoice,
 } from "../../../../common-data/us-state-choices";
+import { getNorentMetadataForUSState } from "./national-metadata";
+import { OutboundLink } from "../../analytics/google-analytics";
 
 export const NorentLbKnowYourRights = MiddleProgressStep((props) => {
   const { session } = useContext(AppContext);
   const state = session.norentScaffolding?.state as USStateChoice;
```

Comment:
To clarify, it _is_ ok to use `norentScaffolding.state` here because the user doesn't actually have an account yet, so `norentScaffolding.state` is the temporary place we've put that information they just entered.